front-end/app/upload_tutorial/page.tsx

```diff
@@ -1,55 +1,46 @@
-            <p>Easily see only the information you need in an eCR</p>
-            <ProcessList>
-                <ProcessListItem>
-                    <ProcessListHeading type="h4">Start a process</ProcessListHeading>
-                    <p className="margin-top-05">
-                        Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo,
-                        ipsum sed pharetra gravida, orci magna rhoncus neque.
-                    </p>
-                    <ul>
-                        <li>
-                            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi
-                            commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id
-                            pulvinar odio lorem non turpis.
-                        </li>
-                        <li>
-                            Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat
-                            condimentum.
-                        </li>
-                        <li>Aliquam erat volutpat. Sed quis velit.</li>
-                    </ul>
-                </ProcessListItem>
-                <ProcessListItem>
-                    <ProcessListHeading type="h4">
-                        Proceed to the second step
-                    </ProcessListHeading>
-                    <p>
-                        Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo,
-                        ipsum sed pharetra gravida, orci magna rhoncus neque, id pulvinar odio
-                        lorem non turpis. Nullam sit amet enim. Suspendisse id velit vitae
-                        ligula volutpat condimentum. Aliquam erat volutpat. Sed quis velit.
-                        Nulla facilisi. Nulla libero. Vivamus pharetra posuere sapien.
-                    </p>
-                </ProcessListItem>
-                <ProcessListItem>
-                    <ProcessListHeading type="h4">
-                        Complete the step-by-step process
-                    </ProcessListHeading>
-                    <p>
-                        Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat
-                        condimentum. Aliquam erat volutpat. Sed quis velit. Nulla facilisi.
-                        Nulla libero. Vivamus pharetra posuere sapien.
-                    </p>
-                </ProcessListItem>
-            </ProcessList>
-            <Button type="button">Get Started</Button>
+        <div className="display-flex flex-justify-center margin-top-5">
+            <div>
+                <h1 className="font-sans-2xl text-bold">eCR Viewer Tool</h1>
+                <h2 className="font-sans-lg text-light">Easily find the information you need from an eCR</h2>
+                <ProcessList className='padding-top-4'>
+                    <ProcessListItem>
+                        <ProcessListHeading type="h4">Upload your eCR</ProcessListHeading>
```

Comment:
Shouldn't the `<ProcessListHeading`s be `type="h3"`? Otherwise, you're skipping `h3`. I suppose this is not a public-facing site, so we don't need to be as concerned with accessibility, but this can mess with screen readers.